src/Utils/Hooks/useWindowSize.ts

```diff
@@ -2,18 +2,22 @@ import { useLayoutEffect, useState } from "react"
 import { getViewportDimensions } from "Utils/viewport"
 
 export const useWindowSize = () => {
-  const [size, setSize] = useState({ width: 0, height: 0 })
+  if (typeof window !== "undefined") {
```

Comment:
I agree with @damassi that returning early makes this code more readable, particularly in this case because there is _something happening_ in the alternate branch. If we were just not doing anything if `window` was `undefined`, then I would say it's a horse a piece; in this case I have to scan pretty far down to see that we're actually returning a defaulted object.

@ashleyjelks would you mind making these changes?